### PR TITLE
Update Sensor Configuration Files

### PR DESCRIPTION
**Pull request by levobot**
Update Sensor configuration files.

### CODE REVIEW

File: static/artifacts/sensor/config-values.yml [hunk 1]
```diff
@@ -165,3 +165,34 @@ sensor:
     #
     
     # --------------------------------------------------------------------------------------------
+
+    # --------------------------------------------------------------------------------------------
```
Comment (hnataraj) [inline static/artifacts/sensor/config-values.yml:169]: @ricekot Can we move this to the factory settings section? Or does it complicate our process of generating the yml files for different environments ?

Ideally these will belong in a single factory settings section at the very top.

File: static/artifacts/sensor/config-values.yml [hunk 2]
```diff
@@ -165,3 +165,34 @@ sensor:
...
+    # Pod Filters enable granular capture of API traffic based on Kubernetes Pod attributes.
+    # Rules should ideally be in decreasing order of specificity.
+    # The first rule to match a pod's properties will be used.
+    # --------------------------------------------------------------------------------------------
+    #
+    k8s-pod-filter-list:
+      default-policy: trace
+      rules:
+        - policy: ignore
```
Comment (hnataraj) [inline static/artifacts/sensor/config-values.yml:187]: @ricekot could we add comments that these are default settings meant to ignore system level traffic?

File: static/artifacts/sensor/config-values.yml (hunk 1 above)
Comment (mprzybylski) [reply static/artifacts/sensor/config-values.yml:176]: I'm OK with moving it up to the main factory settings section with an explanatory comment.  I'm doing so now.

File: static/artifacts/sensor/config-values.yml (hunk 2 above)
Comment (mprzybylski) [reply static/artifacts/sensor/config-values.yml:194]: Done.

**Comment by mprzybylski** [comment]
Closing this PR because we have an incoming merge that is about to generate another one with @hnataraj's comments addressed.

**Pull request closed without merging**